cms/djangoapps/contentstore/tasks.py

```diff
@@ -202,12 +202,15 @@ def async_export_to_git(course_module, user=None):
     Exports a course to Git.
     """
     try:
+        LOGGER.debug('Starting async course content export to git (course id: %s)', course_module.id)
```

Comment:
If we want this to execute in QA or production then it will need to be INFO level. The edX code is written in a way that prevents debug logging on anything other than a developer machine.

Reply:
Yeah, I didn't know if we wanted to make this an `INFO` level. The message below it was `DEBUG` so I just followed the "convention". If you don't think it would be too noisy I'm happy to switch them

Reply:
It's fine to keep it as DEBUG, it just won't be visible in the QA or production environments, so it's dependent on the level of utility that you are hoping to gain from those debug messages.